external/anomaly/tests/conftest.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+from ote_sdk.test_suite.pytest_insertions import *
+from ote_sdk.test_suite.training_tests_common import REALLIFE_USECASE_CONSTANT
+
+pytest_plugins = get_pytest_plugins_from_ote()
+
+ote_conftest_insertion(default_repository_name='ote/training_extensions/external/anomaly')
+
+@pytest.fixture
+def ote_test_domain_fx():
+    return 'custom-anomaly-classification'
```

Comment:
Now that we already have anomaly segmentation in the repo, and detection in a PR stage. I guess there will be an upcoming PR for those?

Reply:
Yep, if needed, it can be changed in later PR's